.github/dependabot.yml

```diff
@@ -8,4 +8,8 @@ updates:
     day: "sunday"
     time: "04:00"
   open-pull-requests-limit: 30
-  versioning-strategy: increase
\ No newline at end of file
+  versioning-strategy: increase
+  ignore:
+    # ignore all GitHub linguist patch updates
+    - dependency-name: "github-linguist"
```

Comment:
Hvis jeg tolker dette riktig gjør denne at vi bare ignorer patcher for pakken `github-linguist`, og den har vi vel ikke? 🤔 

Kanskje vi vil ignore patcher for alle pakker?

Reply:
ahaaa :) Smart :p